JASP-Desktop/widgets/listmodeltableviewbase.cpp

```diff
@@ -45,7 +45,7 @@ QVariant ListModelTableViewBase::data(const QModelIndex &index, int role) const
 	int		column	= index.column(),
 			row		= index.row();
 
-	if (column < 0 || column >= columnCount() || row < 0 || row >= _rowNames.length())
+	if (column < 0 || column >= columnCount() || row < 0 || row >= rowCount())
```

Comment:
It is surely better, but I still not understand why it solves the problem

Reply:
This isnt what solves the problem actually. Ill comment at the line where I did.